### PR TITLE
Add recipe for Neper

### PR DESCRIPTION
**Pull request by npavlovikj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by npavlovikj** [comment]
1. I needed to add `libblas` and `llvm-openmp` to `host`, otherwise "["  ERROR (neper,bin/neper): .. but ['libopenblas'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)"]" and "["  ERROR (neper,bin/neper): .. but ['llvm-openmp'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)"]" errors were produced.
I tried adding `libblas` and `blas` to `ignore_run_exports`, but the error remained.

2. `neper` uses few other packages from "https://github.com/rquey/neper/tree/devel/src/contrib". They are all open source and allow re-distribution, and the licenses are summarized in "https://github.com/rquey/neper/blob/devel/CREDITS", which is listed as an additional `license_file` here.

3. `neper` is only supported on Linux systems, thus Windows is skipped.

@conda-forge/help-c-cpp , this recipe is ready for review. Please see the notes above, and let me know if any additional modifications or clarifications are needed.

File: recipes/neper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - openmp
+    - gmsh
+
+test:
+  commands:
+    - neper --help
+
+about:
+  home: http://neper.info/
+  license: GPL-3
```
Comment (ocefpaf) [inline recipes/neper/meta.yaml]: I believe that the SPDX form is GPL-3.0 but that can be fixed later in the feedstock.

Comment (npavlovikj) [reply recipes/neper/meta.yaml]: I initially had `license: GPL-3.0`, that's why I switched to `license: GPL-3` based on the information in conda-build. From skimming over the other PRs, `license: GPL-3.0-or-later` may the right one.

**Comment by npavlovikj** [comment]
Thank you for the review and merge @ocefpaf !

File: recipes/neper/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/neper/meta.yaml]: You are right. It should be `GPL-3.0-or-later`.

Comment (npavlovikj) [reply recipes/neper/meta.yaml]: As you suggested, I can easily fix that in the feedstock later.

**Comment by npavlovikj** [comment]
On related note, the travis CI build for this this PR, https://travis-ci.com/github/conda-forge/staged-recipes, failed with:
```
Traceback (most recent call last):
870  File ".travis_scripts/create_feedstocks.py", line 202, in <module>
871    if repo_exists(gh, 'conda-forge', name + '-feedstock'):
872  File ".travis_scripts/create_feedstocks.py", line 61, in repo_exists
873    org = gh.get_organization(organization)
874  File "/home/travis/miniconda/lib/python3.7/site-packages/github/MainClass.py", line 263, in get_organization
875    "/orgs/" + login
876  File "/home/travis/miniconda/lib/python3.7/site-packages/github/Requester.py", line 275, in requestJsonAndCheck
877    return self.__check(*self.requestJson(verb, url, parameters, headers, input, self.__customConnection(url)))
878  File "/home/travis/miniconda/lib/python3.7/site-packages/github/Requester.py", line 286, in __check
879    raise self.__createException(status, responseHeaders, output)
880github.GithubException.RateLimitExceededException: 403 {'message': 'API rate limit exceeded for user ID 37169284.', 'documentation_url': 'https://developer.github.com/v3/#rate-limiting'}
881+++set +x
882+++python .travis_scripts/trigger_travis_build.py conda-forge/staged-recipes
883b'access denied'
884Traceback (most recent call last):
885  File ".travis_scripts/trigger_travis_build.py", line 54, in <module>
886    sys.exit(main(sys.argv))
887  File ".travis_scripts/trigger_travis_build.py", line 47, in main
888    rebuild_travis(args.slug)
889  File ".travis_scripts/trigger_travis_build.py", line 38, in rebuild_travis
890    response.raise_for_status()
891  File "/home/travis/miniconda/lib/python3.7/site-packages/requests/models.py", line 940, in raise_for_status
892    raise HTTPError(http_error_msg, response=self)
893requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.travis-ci.com/repo/conda-forge%2Fstaged-recipes/requests
```
:/

**Comment by npavlovikj** [comment]
Please ignore my comment earlier - I forgot that one feedstock can be converted per hour. I restarted the build again and the feedstock was generated.

**Pull request merged**